pkg/driver/controller_modify_volume.go

```diff
@@ -39,11 +40,16 @@ const (
 	ModificationKeyIOPS = "iops"
 
 	ModificationKeyThroughput = "throughput"
+
+	ModificationAddTag = "tagSpecification"
+
+	ModificationDeleteTag = "tagDeletion"
 )
 
 type modifyVolumeRequest struct {
 	newSize           int64
 	modifyDiskOptions cloud.ModifyDiskOptions
+	modifyTagsOptions *cloud.ModifyTagsOptions
```

Comment:
I think now that #2083 is merged this doesn't need to be a pointer anymore (make sure you rebase on top of `master`)